app/controllers/restaurants_controller.rb

```diff
@@ -14,7 +14,8 @@ def show
   # POST /restaurants
   def create
     # create! RecordInvalid exception handled in ApplicationController
-    Restaurant.create!(restaurant_params)
+    rest = Restaurant.create!(restaurant_params)
```

Comment:
`create!`, `update!`, `destroy!` and all the bang function will throw an exception, and all the "matching" function (without the bang) will not throw an exception. This is a convention on Ruby on Rails :)

Reply:
I've implemented exception handling for these scenarios in the base class (ApplicationContoller) instead of null checks

Reply:
saw the comment, just a convention FYI :)